modules/passkey/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@safe-global/safe-passkey",
+  "version": "0.1.0",
+  "author": "@safe-global",
+  "description": "Safe Passkey Owner",
+  "homepage": "https://github.com/safe-global/safe-modules/4337",
```

Comment:
I'd just link the safe-modules repo without nested paths

Reply:
Personally, I like linking to the subdirectory, it makes navigation from NPM more seamless.

Will just fix the path though.